### PR TITLE
mongodb[major]: Create MongoDB integration package

### PR DESCRIPTION
**Pull request by jacoblee93**

### CODE REVIEW

File: libs/langchain-community/package.json [hunk 1]
```diff
@@ -229,7 +229,7 @@
     "llmonitor": "^0.5.9",
     "lodash": "^4.17.21",
     "lunary": "^0.6.11",
-    "mongodb": "^5.2.0",
+    "mongodb": "*",
```
[the remark this answers is not in the document]
Comment (jacoblee93) [reply libs/langchain-community/package.json:232]: Yeah can switch - rationale is anyone currently using this has a lockfile and anyone new would use the new package

**Comment by jacoblee93** [comment]
Thank you for the review!

**Comment by baileympearson** [comment]
@jacoblee93 Is there anything that I can do to help this PR along?  I'm happy to make the proposed changes if that's helpful.

**Comment by jacoblee93** [comment]
> @jacoblee93 Is there anything that I can do to help this PR along? I'm happy to make the proposed changes if that's helpful.

Hey! No, just got busy. Spoke with a few folks on the Mongo team today and we're going to shoot for an announcement in a few weeks, but we could get this merged sooner for sure.

**Comment by prakul** [comment]
@jacoblee93 If there is consensus from @baileympearson, let's get this PR merged. The announcement of this could be done separately, as you suggested.

**Pull request merged**